ldk/node/src/jest.helpers.ts

```diff
@@ -0,0 +1,89 @@
+import {
```

Comment:
Not sure this is the best place for this file. I'm not a huge fan of DRYing up tests if it means they are less readable. However, these seemed like reasonable utility functions to me.

Reply:
I agree with you both on the principle and the judgment call here!